python_ta/.pylintrc

```diff
@@ -20,6 +20,18 @@ pyta-template-file = template.html
 # Set the output file name for HTMLreporter.
 pyta-output-file = pyta_report.html
 
+# Set whether you wish to opt-in to anonymous data submission.
```

Comment:
Change this to read: "Set whether you wish to opt-in to anonymous data collection of errors reported when you run PyTA." (And remove the line after this one.)